08/model.py

```diff
@@ -0,0 +1,82 @@
+import controller
+import interface
+import pandas
+import logger
+
+
+def menu_choice(data):
+    logger.logging.info("Вывод меню")
+    if data == '1':
+        view_file_csv()
+    elif data == '2':
+        add_row_in_file_csv()
+    elif data == '3':
+        change_row_in_file_csv()
+    elif data == '4':
+        delete_row_in_file_csv()
+    elif data == '5':
+        create_file_csv()
+    elif data == 'x':
+        exit()
+    else:
+        print('Выбрано не верное значения, повторите ввод!')
+        controller.run()
+
+
+def view_file_csv():
+    logger.logging.info("Просмотр базы данных")
+    data = pandas.read_csv('db.csv')
+    interface.view_data(data)
+
+
+def add_row_in_file_csv():
+    data = pandas.read_csv('db.csv')
+    last_id = data['id'].loc[data.index[-1]] + 1
```

Comment:
```suggestion
    last_id = len(data)
```
Но если бы у вас начиналась индексация с нуля труда бы вычли еще `1`

Reply:
Это скорее индивидуальный номер, идет по порядку, поэтому 0 не очень использовать. 
Заполняется автоматически чтобы дублей не было.